### PR TITLE
SPP-8500-Accessibility-Page

### PR DESCRIPTION
**Pull request by sanjeevz3009**
# Description

Accessibility statement page:
- Added the route for the accessibility page
- Accessibility statement content is added to a markdown file which is taken and rendered onto the accessibility statement HTML template

About page:
- The variable assigned within the render_template method to be used in the about page HTML template was misspelled, so corrected from "page_boby" to "page_body"
- Updated the about page HTML template to reflect the variable name correction

Added generic BDD tests for all the pages within the footer section:
- Cookies page
- Accessibility Statement page
- Privacy and Data Protection Page

Extracted the _page_not_found function which is used to display a 404 status on the web page and made it into a utility function that can now be reused across the web app.

Added further error handling to the codebase:
- Glossary page
- Help centre page
- Methods catalogue page
- About page
- Security page

Code commenting & documentation are not necessary in this case.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Technical Enhancements

# Checklist:

If any of these are not completed, please explain why in the notes.

## **Definition of Done**
**Code and merges**

- [ ] Code to be commented on where applicable 
- [ ] Documentation updated where required 
- [x] Have considered non-functional requirements such as Security, Performance, Scalability, and Fault Tolerance
- [x] I have linted the code

**Testing**

- [x] All levels of acceptance test are passing (automated, integration, manual, accessibility, etc.)
- [x] I have run the behave command to check the selenium behaviour tests pass locally
- [x] Acceptance criteria met

**Other Checks**
- [x] I have performed a self-review of my own code
- [x] I have checked for spelling errors
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have added the JIRA ticket link to this PR
- [x] My changes don't break anything unexpected
- [x] I have checked and updated the security.txt file where required
- [x] Up to date with the main branch

### CODE REVIEW

File: features/footer.feature [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Footer feature tests
+
+Feature: Footer pages tests
```
Comment (jhmjam) [inline features/footer.feature:3]: I like this :)

File: sml_builder/page.py [hunk 2]
```diff
@@ -27,6 +27,20 @@ def cookies_page():
...
+def accessibility_page():
+    try:
+        with open(
+            "./content/accessibility/accessibility-statement.md", "r", encoding="utf-8"
+        ) as input_file:
+            text = input_file.read()
+            escaped_text = escape(text)
+            body = Markup(markdown.markdown(escaped_text))
+    except OSError as e:
+        _page_not_found(e)
```
Comment (jhmjam) [inline sml_builder/page.py:40]: quick question is this a specific function?

File: content/accessibility/accessibility-statement.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# Accessibility statement
+
+This accessibility statement applies to the Office for National Statistics (ONS) Statistical Methods Library.  
+
+We want as many people as possible to be able to use the website. If you have a disability, [AbilityNet](https://mcmw.abilitynet.org.uk/) has advice on making your device easier to use.
```
Comment (jhmjam) [inline content/accessibility/accessibility-statement.md:5]: space needed 
image

**Review by jhmjam — CHANGES_REQUESTED** [review]

**Comment by jhmjam** [comment]
Good work :)

File: sml_builder/page.py (hunk 2 above)
Comment (sanjeevz3009) [reply sml_builder/page.py:40]: Yes, it's a function that we have. We call it to display a 404 page not found.

File: content/accessibility/accessibility-statement.md (hunk 3 above)
Comment (sanjeevz3009) [reply content/accessibility/accessibility-statement.md]: Done =)

File: sml_builder/page.py (hunk 2 above)
Comment (jhmjam) [reply sml_builder/page.py:40]: Nice one thanks

File: content/accessibility/accessibility-statement.md (hunk 3 above)
Comment (jhmjam) [reply content/accessibility/accessibility-statement.md]: Nice

**Review by jhmjam — APPROVED** [review]

**Pull request merged**